pkg/controller/appdefinition/namespace.go

```diff
@@ -82,3 +83,24 @@ func RequireNamespace(h router.Handler) router.Handler {
 		return h.Handle(req, resp)
 	})
 }
+
+func BackPopulateProjectLabel(req router.Request, resp router.Response) error {
```

Comment:
This name makes it sound like we are back-populating labels from another object to the namespace. I like `AddAcornProjectLabel` better.

What do you think?

Reply:
Yeah that sounds better to me as well